edudoc/delegate-handbook/delegate-handbook.md

```diff
@@ -1289,12 +1232,12 @@ Further disagreements with the result of a dispute may be escalated to an appeal
 
 There may be a time when you want to make a member of your community a Delegate. Some reasons a new Delegate may be needed are to alleviate the workload of existing Delegates or to provide a more local Delegate who can better serve the needs of a remote cubing community. However, there are no hard and fast rules about how many Delegates are required for a given region, or where Delegates should be located. Discuss with any other local, Regional, or Senior Delegate whether adding a new Delegate is an appropriate step for your community.
 
-To begin the process of inducting a member of your community as a Delegate, follow the following steps:
+To begin the process of inducting a member of your community as a Delegate, follow these steps:
```

Comment:
I feel like it's said "the following" earlier

Reply:
I know, that's what I changed. "Follow the following" just sounded a bit clunky.

Reply:
Oh, I see, line 526 has "take the following steps". Should I change 1235 to "take the following steps" too?

Reply:
I think that's a good idea!